### ISSUE TITLE
How can I more UI elements to view

### ISSUE THREAD

**Issue by jgutierrez11** [zhxnlai/ZLSwipeableViewSwift#60]
Hi, I am trying to use this library but I can't manage to add labels and other UI elements to the swipeable View... does anyone has a demo project where it uses something similar?

thanks!

**Comment by AndrewSB** [comment]
@jgutierrez11 can you share the code where you're trying to add the UI elements?

**Comment by jgutierrez11** [comment]
I have added two labels to the CardContentView. Then I created IBOutlets for both of them.
I don't know how to override the init to pass the data to these controls. If I try to add them to the overriden init like so: **override init(frame: CGRect, label1: UILabel)** it says: **Initializer does not override a designated initializer from its superclass** if i try to access the label thru the func setup:       label.text = "Hello world" it says: **fatal error: unexpectedly found nil while unwrapping an Optional value** so I really have no clue on how to add the custom controls and tie them to the code.
